documents/implementation-guide.md

```diff
@@ -93,9 +93,9 @@ Usage example:
 ```js
 import { localize, Localize } from 'components/localization'
 
-<p>{localize('text need to be translated')}</p>
+<p>{localize('_t_text need to be translated_t_')}</p>
```

Comment:
We have to provide current approach in guide as well

<img width="882" alt="Screenshot 2023-03-13 at 10 30 02" src="https://user-images.githubusercontent.com/103182473/224635081-d23dd7f3-97e7-4a59-ade8-49dc48c4cfb0.png">